### ISSUE TITLE
iTerm2 v3 broke Applescript

### ISSUE THREAD

**Issue by pjaspers** [pjaspers/wasko#5]
> The good news is that iTerm2's Applescript makes a lot more sense than it used to. You can manipulate split panes, and scripting will receive more love and support in the future.
> 
> The bad news is that existing Applescript won't work any more.

This should have all the info to fix it: http://iterm2.com/documentation-scripting.html

**Comment by bramj** [comment]
Is this what causes `syntax error: A identifier can’t go after this identifier.`?

**Comment by pjaspers** [comment]
Yep, I've got top men on the [case](https://github.com/pjaspers/wasko/pull/4). Top. Men.
